graylog2-web-interface/src/components/event-notifications/event-notification-types/EmailNotificationForm.jsx

```diff
@@ -95,7 +95,7 @@ class EmailNotificationForm extends React.Component {
     body_template: DEFAULT_BODY_TEMPLATE, // TODO: Default body template should come from the server
     html_body_template: DEFAULT_HTML_BODY_TEMPLATE,
     user_recipients: [],
-    email_recipients: [],
```

Comment:
Since the time zone value defaults to `UTC`, shouldn't we add the default value here so it shows in the form? That way it is clearer what happens if you don't set a time zone.